Source/WebCore/platform/graphics/NativeImage.cpp

```diff
@@ -49,4 +49,10 @@ NativeImage::~NativeImage()
         observer->releaseNativeImage(m_renderingResourceIdentifier);
 }
 
+void NativeImage::setPlatformImage(PlatformImagePtr&& platformImage)
+{
+    ASSERT(m_platformImage);
```

Comment:
Is this right?

Reply:
No it is not right. I think we should assert the argument `platformImage` is not null.